dexcom/event.go

```diff
@@ -95,6 +95,12 @@ func (e *Event) validateCarbs(validator structure.Validator) {
 }
 
 func (e *Event) validateExercise(validator structure.Validator) {
+	// HACK: Dexcom - value of -1 is invalid; ignore unit and value instead (per Dexcom)
+	if e.Value != nil && *e.Value == -1.0 {
```

Comment:
What does `*e` do?

Reply:
`e` is an "object" that has a field `Value` that is a pointer to a `float64`. So, the `e.Value != nil` ensures that the pointer is not nil and `*e.Value == -1.0` ensures that the `float64` the pointer points to is equal to `-1.0`.

So, in short, the `*` is dereferencing the `e.Value`.

Reply:
Ah, makes sense.